components/eventing-controller/controllers/subscription/nats/unit_test.go

```diff
@@ -29,81 +30,84 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
-const namespaceName = "test"
+const (
+	subscriptionName = "testSubscription"
+	namespaceName    = "test"
+)
 
 var (
 	defaultSubsConfig = env.DefaultSubscriptionConfig{MaxInFlightMessages: 1, DispatcherRetryPeriod: time.Second, DispatcherMaxRetries: 1}
 	testEnv           *envtest.Environment
```

Comment:
is this really needed?

Reply:
Yes, because I want to use constants instead of magic strings

Reply:
I see now, that you mentioned the `testEnv` variable, it is removed now